src/core/helpers.js

```diff
@@ -72,6 +72,60 @@ wb.download = function( blob, filename, title ) {
 
 };
 
+/* ---------------------------------
+@extension: shuffleDOM
+@returns: [list] shuffles a list of items randomly
+-------------------------------- */
+wb.shuffleDOM = function( DOM ) {
+	var allElems = DOM.get(),
+		shuffled = $.map( allElems, function() {
+			var random = Math.floor( Math.random() * allElems.length ),
+				randEl = $( allElems[ random ] ).clone( true )[ 0 ];
+			allElems.splice( random, 1 );
+			return randEl;
+		} ),
+		DOMLen = DOM.length;
+
+	for ( var i = 0; i < DOMLen; i++ ) {
+		$( DOM[ i ] ).replaceWith( $( shuffled[ i ] ) );
+	}
+
+	return $( shuffled );
+};
+
+/* ---------------------------------
+@extension: pickElements
+@returns: [collection] of randoms elements
+-------------------------------- */
+wb.pickElements = function( DOM, k ) {
```

Comment:
Can you try to unminify the code in this function? Like try to put more descriptive variable name.

Like changing `DOM` for `elm` for consistency with the rest of WET code that make reference to an DOM element or for `$elm` if `DOM` is a jQuery object.

This assignation have no signifiant value: `b = DOM`